.github/workflows/e2e-suite.yml

```diff
@@ -108,11 +112,23 @@ jobs:
         run: |
           echo "LINODE_CLI_TOKEN=${{ secrets[inputs.use_minimal_test_account == 'true' && 'MINIMAL_LINODE_TOKEN' || 'LINODE_TOKEN'] }}" >> $GITHUB_ENV
 
+      - name: Set OpenAPI Spec URL
+        if: ${{ inputs.openapi_spec_url != '' }}
+        run: |
+          echo "Using OpenAPI Spec URL: ${{ inputs.openapi_spec_url }}"
+          export OPENAPI_SPEC_URL="${{ inputs.openapi_spec_url }}"
```

Comment:
Looks great so far!
Could be used with the `SPEC=...` argument in the make install step above?